CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ To make things easier we've created these guidelines:
 Unit tests can be run locally. Please see our [ test instructions ](https://github.com/woothemes/sensei/tree/master/tests#readme) to run the unit tests.
 
 ## Development Blog
-Please follow further development updates at [develop.woothemes.com/sensei]( http://develop.woothemes.com/sensei )
+Please follow further development updates at [develop.woothemes.com/sensei]( http://senseilms.wordpress.com/ )
```

Comment:
This should be http**s**://senseilms.wordpress.com so we can avoid a redirect.

Reply:
Actually, we've just launched a new site. The URL should now be https://senseilms.com/blog.